src/frontend/images/mdi/README.md

```diff
@@ -0,0 +1,22 @@
+## Icons
+To use Material Design icons.
+
+- Download 'mdi.svg' from here [1][1].
```

Comment:
I do not understand why we need to download?

Reply:
I copy it from the old codebase.